react-app/src/components/PatientData.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import ReactJson from 'react-json-view';
+import LogList from './LogList';
 
 function PatientData(props) {
   return (
     <div>
       {props.id === null && (
         <div className="page-text">
-          <p>Select a patient to view their information.</p>
+          <p>Select a patient to view their information or click on "Log File" to view the logger messages.</p>
         </div>
       )}
-      {props.id !== null && (
+      {props.id !== null && props.id === -1 && (
+        <div>
+          <h3 className="page-subtitle sticky-item">Log File</h3>
+          <LogList loggedMessages={props.loggedMessages} />
+        </div>
+      )}
+      {props.id !== null && props.id >= 0 && (
```

Comment:
Per my comments on the previous PR, I think we should try to avoid using -1 here for the patient ID.

Really what the -1 is representing here is a different piece of information, which is that we wish to view the Logs rather than any patient data. I think this should be controlled in a separate state variable, something like `[showLogs, setShowLogs]` which is a boolean that indicates whether we are viewing the logs instead of patient data. Let me know if you have questions about this. There is one other comment I'll make as a companion to this one